webpack.config.js

```diff
@@ -37,9 +86,24 @@ module.exports = {
   plugins: [
     new ThreadsPlugin({
       globalObject: false
+    }),
+    new CopyPlugin({
+      patterns: [
+        {
+          from: 'src/**/*.proto',
+          to: '',
+          flatten: true
+        },
+      ]
+    }),
+    new NpmDtsPlugin({
```

Comment:
What's this for? And if this is what I think it is for, why wasn't this used in js-encryptedfs?

Reply:
This was for creating a single definition file `index.d.ts` to simplify things but I don't think it really works, I have taken it out.

Reply:
This was also an attempt at trying to generate normal typings with the tsconfig paths [issue](https://github.com/MatrixAI/Typescript-Demo-Lib/issues/1) which made it much harder since we have a CLI and a LIB in one repo. I still would rather just forget about this as it's been a huge effort to try and integrate it into our project for what seems like minimal benefit.

Reply:
now if I put `"declaration": true` into `tsconfig.json`, it generates typings for the test folder as well, since I've had to 'include' the test folder in order to get `@` imports working from the test folder.
https://github.com/MatrixAI/Typescript-Demo-Lib/issues/1#issuecomment-643945531

Reply:
Hmm the CLI stuff shouldn't generate any kind of typings since it's not meant to be used as a library. It shouldn't even be importable at all.

Perhaps the CLI portion of the code should be moved out into its own directory? That is outside of the `src` directory completely.

But yea if you cannot use `@polykey` in the CLI code src, then so be it.